uw-frame-components/portal/misc/partials/add-to-home.html

```diff
@@ -1,42 +1,33 @@
-<!-- ADD MORE TO HOME BUTTON -->
-<div ng-if='actionLinkUrl && !addToHome'>
-  <md-button class="md-primary link-div" ng-href="{{actionLinkUrl}}" hide-sm hide-xs>
-    <span><i class="fa fa-fw {{actionLinkIcon}}"></i> {{actionLinkText}}</span>
-  </md-button>
-  <md-menu-item hide-gt-sm>
-    <md-button class="md-primary link-div" ng-href="{{actionLinkUrl}}">
-      <span><i class="fa fa-fw {{actionLinkIcon}}"></i> {{actionLinkText}}</span>
-    </md-button>
-  </md-menu-item>
-</div>
-
-<!-- ADD -THIS- TO HOME BUTTON -->
-<div ng-controller="AddToHomeController" ng-if="addToHome">
-  <!-- Button for medium+ screens -->
-  <md-button class="md-default link-div" aria-label="add to home" hide-xs hide-sm ng-hide='inHome' ng-click='addToHome()'>
-    <span ng-if="!successfullyAdded || !addToHomeFailed">
-      <i class="fa fa-fw {{actionLinkIcon}}" ng-class="{ 'fa-spin' : savingAddToHome, 'fa-plus' : !actionLinkIcon}"></i> {{actionLinkText}}
-    </span>
-    <span ng-show='successfullyAdded'>
-      <i class="fa fa-fw fa-check-square"></i> {{actionLinkText}} Saved
-    </span>
-    <span ng-show='addToHomeFailed'>
-      <i class="fa fa-fw fa-exclamation-triangle"></i> {{actionLinkText}} Failed
-    </span>
+<!-- ADD TO HOME BUTTON -->
+<div ng-controller="AddToHomeController" class="animate-add-to-home" ng-hide="inHome">
+  <!-- Buttons for medium+ screens -->
+  <md-button class="md-default link-div" aria-label="add to home" hide-xs hide-sm ng-click="addToHome()" ng-if="!successfullyAdded && !addToHomeFailed && !savingAddToHome">
+    <md-icon ng-hide="savingAddToHome">add</md-icon> Add to home
+  </md-button>
+  <md-button class="md-default link-div" aria-label="adding to home" hide-xs hide-sm ng-click="addToHome()" ng-if="!successfullyAdded && !addToHomeFailed && savingAddToHome">
```

Comment:
Should this button do anything when clicked? ( `ng-click="addToHome()"` ).  Should it be disabled since not meaningfully clickable while adding to home attempt is pending?